<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewStudyOverviewPage</class>
 <widget class="QWidget" name="NewStudyOverviewPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="6" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="7">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Solution Strategie</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="l_SolutionStategy">
        <property name="text">
         <string>SOLUTION NAME</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="l_r_n">
        <property name="text">
         <string>r/n</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="l_enp0">
        <property name="text">
         <string>EN(p0)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="l_petP0">
        <property name="text">
         <string>PET(p0)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="l_alpha">
        <property name="text">
         <string>Alpha</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="l_beta">
        <property name="text">
         <string>Beta</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="l_r1_n1">
        <property name="text">
         <string>r1/n1</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="6">
    <widget class="QGroupBox" name="gB_flexibleOverview">
     <property name="title">
      <string>Flexible extendition</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="l_distributeMethod">
        <property name="text">
         <string>Distribution method</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="t_flexible_Overview">
        <column>
         <property name="text">
          <string>Responses</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>CE</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>General Informtion</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="l_StudyName">
        <property name="text">
         <string>STUDYNAME</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="l_Investigator">
        <property name="text">
         <string>INVESTIGATOR</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="l_Biometrician">
        <property name="text">
         <string>BIOMETRICIAN</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="l_StudyType">
        <property name="text">
         <string>STUDYTYPE: Siomedesign</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QGroupBox" name="gB_Curtailment">
     <property name="title">
      <string>Curtailment</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="2" column="1">
       <widget class="QLabel" name="l_betaSC">
        <property name="text">
         <string>Beta_sc</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="l_enp0SC">
        <property name="text">
         <string>EN(p0)_SC</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="l_alphaSC">
        <property name="text">
         <string>Alpha_sc</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="l_cut">
        <property name="text">
         <string>CurtailmentCut</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="l_petP0SC">
        <property name="text">
         <string>PET(p0)_SC</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="6" rowspan="3" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Graphical overview</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="Rwidget" name="rw_overview" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Rwidget</class>
   <extends>QWidget</extends>
   <header location="global">rwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
